fixtures/framework/README.md

```diff
@@ -9,59 +9,77 @@ The following tools should already be installed
 1. docker
 2. docker-compose
 
-### Starting STRATO
 
-Start STRATO using the following environment variables
+### Obtaining an ADMIN TOKEN
+
+This project requires an `ADMIN_TOKEN` environment varibale for the app deployment to work. This token can be obtained by using the `token-getter` utility packaged in `blockapps-rest`. To execute this utility, run `sudo yarn token-getter` from the `<dir>-server` directory:
 
 ```
-OAUTH_ENABLED=true OAUTH_CLIENT_ID=<client-id> OAUTH_CLIENT_SECRET=<client-secret> OAUTH_DISCOVERY_URL=<discovery-url> OAUTH_JWT_USERNAME_PROPERTY=email HTTP_PORT=8080 NODE_HOST=localhost:8080 ./strato.sh --single
+cd <dir>/<dir>-server
+sudo yarn token-getter
 ```
 
-### Obtaining an ADMIN TOKEN
-
-This project requires an `ADMIN_TOKEN` env varibale for the deploy to work. This token can be obtained by using the `token-getter` utility packaged in `blockapps-rest`. This utility can be executed by running the following command:
-
+This command launches a small web server on port 80. 
+- In your brower, go to `http://localhost`
```

Comment:
same, the URL will be the one from redirectUri, the resulting URL is displayed when token-getter is started